inflammation-analysis.py

```diff
@@ -27,7 +27,8 @@ def main(args):
     for filename in InFiles:
```

Comment:
Test standard deviation for two patients.

Reply:
Brilliant, I'll raise this as an issue on my repo for future work when I work out how.